components/Releases/CreateRelease.jsx

```diff
@@ -0,0 +1,176 @@
+import { useState, useEffect } from "react"
+import { useUser, useSupabaseClient } from "@supabase/auth-helpers-react"
+
+const releaseTypes = [
+  { id: 1, text: "LP" },
+  { id: 2, text: "EP" },
+  { id: 3, text: "Single" },
+  { id: 4, text: "Compilation" },
+  { id: 5, text: "Soundtrack" },
+  { id: 6, text: "Choose release type", isDisabled: true },
+]
+
+export default function CreateRelease({ user, setCreateNewRelease }) {
+  const supabase = useSupabaseClient()
+  const [title, setTitle] = useState()
+  const [artist, setArtist] = useState(
+    user.user_metadata.type === "Artist" ? user.user_metadata.name : null
+  )
+  const [label, setLabel] = useState(
+    user.user_metadata.type === "Label" ? user.user_metadata.name : null
+  )
+  const [artwork_url, setArtwork_url] = useState()
+  const [download_url, setDownload_url] = useState()
+  const [page_password, setPage_password] = useState()
+  const [is_password_protected, setIs_password_protected] = useState(false)
```

Comment:
I know this is going to be super confusing, but you should `camelCase` these variable names as per popular JS standards. The `_` casing is common with SQL, PHP, and general database naming convention

Example: `artwork_url` to `artworkUrl`